src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/utils/ContainerJurisdictionPoBoxMapper.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.bulkscan.endtoendtests.utils;
+
+import com.google.common.collect.ImmutableMap;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.helper.Container;
+
+import java.util.Map;
+
+public final class ContainerJurisdictionPoBoxMapper {
+
+    private static final Map<Container, JurisdictionAndPoBox> CONTAINER_MAPPINGS = ImmutableMap
+        .<Container, JurisdictionAndPoBox>builder()
+        .put(Container.BULKSCAN, new JurisdictionAndPoBox(Container.BULKSCAN.name(), "BULKSCANPO"))
+        .put(Container.CMC, new JurisdictionAndPoBox(Container.CMC.name(), "12747"))
+        .put(Container.DIVORCE, new JurisdictionAndPoBox(Container.DIVORCE.name(), "12706"))
+        .put(Container.FINREM, new JurisdictionAndPoBox(Container.DIVORCE.name(), "12746"))
+        .put(Container.PROBATE, new JurisdictionAndPoBox(Container.PROBATE.name(), "12625"))
+        .put(Container.PUBLICLAW, new JurisdictionAndPoBox(Container.PUBLICLAW.name(), "12879"))
+        .put(Container.SSCS, new JurisdictionAndPoBox(Container.SSCS.name(), "12626"))
```

Comment:
if key is the same as the jurisdiction, can this map be simpler? (only hold PO Boxes)

Reply:
not the same. finrem's jurisdiction is divorce

Reply:
BTW, next one will introduce document subtype. had a feeling it'll expand, but was just going for jurisdiction and pobox for now - the most obvious and mandatory for bulkscan component